app/models/division.rb

```diff
@@ -0,0 +1,7 @@
+class Division < ActiveRecord::Base
```

Comment:
We should also add a `name` method here that concatenates the league and region (e.g. "American League West")